crates/sui-framework/sources/coin.move

```diff
@@ -170,9 +170,23 @@ module sui::coin {
 
     // === Entrypoints ===
 
+    /// Mint `amount` of `Coin` and send it to `recipient`. Invokes `mint()`.
+    public entry fun mint_<T>(
```

Comment:
I totally agree with the requirement of having entry functions for these flows (as it will help new Coins to be created in just a few lines of code), but we need to agree on the naming convention. Is the `_` the pattern we're looking for as best practice here or `mint_to_sender` etc? @tnowacki

Reply:
The `_` convention was from when you actually needed a separate function for visibility reasons.

This feels a bit different though since the functionality is actually different. I would advise something like `mint_and_transfer`

Reply:
@tnowacki what do you think about burn_ in thia context?

Reply:
The one where it drops the `u64`? Feels okay. 

I would ask yourself, What would the `and_<verb>` version be? If it is un-interesting, just an `_` is fine. If its something like `and_transfer_to_someone`, you probably want a different name or suffix

Reply:
This is the way!